From: Mohammad Rafi Shaik <mohammad.rafi.shaik@oss.qualcomm.com>
Date: Fri, 23 Jan 2026 15:31:34 +0530
Subject: ucm2: Qualcomm: qcs8300: Remove Fixed channel setting for
 monaco-evk

Remove Fixed channel setting for monaco-evk to support stereo
capture.

Signed-off-by: Mohammad Rafi Shaik <mohammad.rafi.shaik@oss.qualcomm.com>
Origin: upstream, https://github.com/alsa-project/alsa-ucm-conf/commit/ae76df41fcd17bdf812855449cd9cc40b9c919bc
Last-Update: 2026-10-29
---
 ucm2/Qualcomm/qcs8300/monaco-evk/HiFi.conf | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ucm2/Qualcomm/qcs8300/monaco-evk/HiFi.conf b/ucm2/Qualcomm/qcs8300/monaco-evk/HiFi.conf
index 8345129ce..b60a310e6 100644
--- a/ucm2/Qualcomm/qcs8300/monaco-evk/HiFi.conf
+++ b/ucm2/Qualcomm/qcs8300/monaco-evk/HiFi.conf
@@ -25,6 +25,5 @@ SectionDevice."Mic" {
 	Value {
 		CapturePriority 100
 		CapturePCM "hw:${CardId},1"
-		CaptureChannels 1
 	}
 }
